static/styles/zulip.scss

```diff
@@ -2387,6 +2387,9 @@ div.topic_edit_spinner .loading_indicator_spinner {
         width: 100%;
         min-width: 206px;
         box-sizing: border-box;
+        /* Setting resize as none hides the bottom right diagonal box
+           (which even has a background color of its own!). */
+        resize: none !important;
```

Comment:
Hmm, but don't we want to allow resizing when you can actually edit it?  This looks like it changes not just "View Source" but also "Edit".

Reply:
@timabbott I didn't understand fully what you meant. But if you mean does the box resize when you remove/add content, it still does. (See below GIF).
![f2bead8028e437153590ba0c0d4e58e4](https://user-images.githubusercontent.com/30312566/91436872-9453c100-e886-11ea-94b1-3efc67757011.gif)

Reply:
Yes, but you can no longer trag to change the size in that UI, right?

Reply:
Ah yes, dragging is not be possible. I would still argue on whether the drag is really used for anything? For example, we can drag and resize but saving doesn't snap-it to the resized hight. And readibility wise we have vertical scroll.